src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -111,5 +118,45 @@ void getStations() {
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+        // given
+        Map<String, String> params = new HashMap<>();
+        params.put("name", "강남역");
+
+        ExtractableResponse<Response> response = createStation(params);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        assertThat(findAll().body().jsonPath().getList("name", String.class)).hasSize(1);
+
+        // when
+        removeStation(1L);
+
+        assertThat(findAll().body().jsonPath().getList("name", String.class)).hasSize(0);
```

Comment:
```suggestion
        assertThat(findAll().body().jsonPath().getList("name", String.class)).isEmpty();
```
이걸로 대체해보는건 어떨까요? 숫자는 생각보다 눈에 잘 안띄거든요.